src/components/productListing/ProductsContainer.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from "react";
 import axios from "axios";
 import "./ProductsContainer.css";
+import { useCart } from "../../contexts/cart-context";
 import { useProductsFilters } from "../../contexts/filter-context";
```

Comment:
Suggestion: can use barrel export

so that you can directly import from `../../contexts`